library/farmosUtil/farmosUtil.transplanting.js

```diff
@@ -81,7 +81,7 @@ export async function createTransplantingActivityLog(
 export async function getTransplantingActivityLog(activityLogId) {
   const farm = await getFarmOSInstance();
   const results = await farm.log.fetch({
-    filter: { type: 'log--activity', id: activityLogId },
+    filter: { type: 'log--transplanting', id: activityLogId },
   });
   return results.data[0];
 }
```

Comment:
In the `deleteTransplantingLog` function below the `activity` parameter passed to the `farm.log.delete` function should be changed to be `transplanting`.  It seems to work as is, but using `transplanting` here would better communicate what is happening.